### ISSUE TITLE
License is copied from iloveitaly's dotfiles

### ISSUE THREAD

**Issue by nordfjord** [iloveitaly/github-action-nixpacks#1]
Howdy! Was just skimming through the code and noticed that when finding the repository license it seems to be using the github api to read your own dotfiles repository

https://github.com/iloveitaly/github-action-nixpacks/blob/55780f0a2a2702195e4f6cf929f79cbaf33543b5/entrypoint.sh#L40

Is this intentional?

**Comment by iloveitaly** [comment]
nope! I meant to make this dynamic based on the user using the action. PRs welcome!

**Comment by iloveitaly** [comment]
fixed with https://github.com/iloveitaly/github-action-nixpacks/commit/d57c767b0bdc6c8833551b187f206552dca29713
